.github/workflows/build-misc.yml

```diff
@@ -1,7 +1,13 @@
 on:
-  push: []
-  pull_request: []
-  issue_comment: []
```

Comment:
don't you need issue_comment?

Reply:
I didn't think it needed to respond to the issue comments.

Reply:
I was mistaken